cardform/src/main/java/com/mercadolibre/android/cardform/presentation/ui/formentry/IssuersFragment.kt

```diff
@@ -156,12 +157,24 @@ internal class IssuersFragment : InputFragment() {
     inner class IssuerViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
         val radioButton: AppCompatRadioButton = itemView.findViewById(R.id.radioButton)
         private val issuerImage: ImageView = itemView.findViewById(R.id.issuerImage)
+        private val issuerText: TextView = itemView.findViewById(R.id.issuerText)
 
-        fun loadImage(imageUrl: String?) {
+        fun loadImage(imageUrl: String?, name: String) {
             PicassoDiskLoader
                 .get(issuerImage.context)
                 .load(imageUrl)
-                .into(issuerImage)
+                .into(issuerImage, object: com.squareup.picasso.Callback {
+                    override fun onSuccess() {
+                        issuerImage.visibility = View.VISIBLE
+                        issuerText.visibility = View.GONE
```

Comment:
Question here: do we have Android KTX extensions in this project? There is a KTX extension for visibility and can be used as suggested (I know we already have KTX core somewhere)

```suggestion
                        issuerImage.isVisible = true
                        issuerText.isGone = true
```